frontend/pages/index.page.tsx

```diff
@@ -281,3 +262,31 @@ const SubscriptionBox: FC = () => {
     </StyledCard>
   );
 };
+
+export const getStaticProps: GetStaticProps = async () => {
+  let todayinhistoryData = {};
+  let featuredcontentData = {};
```

Comment:
These should start as empty arrays, since they are arrays when set by the api calls.

Reply:
Yup, `{}` --> `[]`.